modules/gcp/rules.tf

```diff
@@ -1,6 +1,7 @@
 locals {
   internal_network_blocks        = ["10.0.0.0/8"]
   bitmovin_static_network_blocks = ["104.199.97.13/32", "35.205.157.162/32"]
+  is_automode_vpc = length(var.enabled_regions) == 0 ? true : false
```

Comment:
If we provide an empty list of regions, we want to enable (which is the variable default), we create the VPC with the auto_create_subnetworks parameter set to true, as we currently do.